pkg/endpoint/policy.go

```diff
@@ -478,12 +478,12 @@ func (e *Endpoint) regenerateConsumable(owner Owner, labelsMap *policy.IdentityC
 	}
 
 	e.getLogger().WithFields(logrus.Fields{
-		logfields.Identity: c.ID,
-		"consumers":        logfields.Repr(c.Consumers),
-		"rulesAdd":         rulesAdd,
-		"l4Rm":             l4Rm,
-		"rulesRm":          rulesRm,
-	}).Debug("New consumable with consumers")
+		logfields.Identity:          c.ID,
+		"ingressSecurityIdentities": logfields.Repr(c.IngressIdentities),
+		"rulesAdd":                  rulesAdd,
+		"l4Rm":                      l4Rm,
+		"rulesRm":                   rulesRm,
+	}).Debug("New consumable with ingress security identities")
```

Comment:
`Consumable regenerated` ? The `with ...` doesn't seem to add much here, the relevant fields are already included in the fields above.

Reply:
Done.